support-frontend/assets/helpers/productPrice/weeklyProductPrice.js

```diff
@@ -5,14 +5,22 @@ import { currencies, detect } from 'helpers/internationalisation/currency';
 import type { WeeklyBillingPeriod } from 'helpers/billingPeriods';
 import { Annual, Quarterly, SixForSix } from 'helpers/billingPeriods';
 import { getPromotionWeeklyProductPrice, getWeeklyProductPrice } from 'helpers/subscriptions';
+import type { Price } from 'helpers/productPrice/productPrices';
 
 
-const getPrice = (countryGroupId: CountryGroupId, period: WeeklyBillingPeriod) => [
+export const getPrice = (countryGroupId: CountryGroupId, period: WeeklyBillingPeriod) => [
   currencies[detect(countryGroupId)].extendedGlyph,
   getWeeklyProductPrice(countryGroupId, period),
 ].join('');
 
-const getPromotionPrice = (countryGroupId: CountryGroupId, period: WeeklyBillingPeriod, promoCode: string) => [
+export const getCurrencyAndPrice = (countryGroupId: CountryGroupId, period: WeeklyBillingPeriod): Price => (
```

Comment:
There's going to have to be some changes to this code because it's using hard coded values in subscriptions.js at the moment and it should be using the prices from the `productPrices` object in the same way as the other two checkouts, but that can be a separate PR.

Reply:
👍